### PR TITLE
Fix: Order success Change the download tickets to all tickets button

### LINKED ISSUE #5852: Order Success Page: Change "Download Tickets" to "All Tickets" and use download icon
On the order success page we have two buttons a) Download Ticket and b) "Download Tickets". This issue is about the button "Download Tickets". Please change the button to "All Tickets" and use a download icon for this specific button.

Screenshot from 2020-12-01 20-57-25

### PR DESCRIPTION
**Pull request by Raunaksingh100**
Fixes #5852 

#### Short description of what this resolves:
Change the icon as well as fixed the button name see in thee screenshot for the better understanding.

### scrennshot
n

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by Raunaksingh100** [comment]
@mariobehling  i perfectly fixed the issue

**Review by cggvuhh — APPROVED** [review]

**Review by harpreetsingh1234 — APPROVED** [review]

**Comment by mariobehling** [comment]
Please keep the position and width of the buttons as before.

**Comment by Raunaksingh100** [comment]
> Please keep the position and width of the buttons as before.

@mariobehling width is adjust by default by the content we can set the width

**Comment by Raunaksingh100** [comment]
> Please keep the position and width of the buttons as before.

@mariobehling we cannot set size of button it automatically increase by the content.

File: app/styles/partials/overrides.scss [hunk 1]
```diff
@@ -84,7 +84,7 @@ body.dimmable.undetached.dimmed {
 }
 
 .space-between {
-  justify-content: space-between;
+  justify-content: space-evenly;
```
Comment (iamareebjamal) [inline app/styles/partials/overrides.scss:87]: You can't just change the class content without impacting everywhere the class is used

Comment (Raunaksingh100) [reply app/styles/partials/overrides.scss:87]: i will fixed this

**Comment by iamareebjamal** [comment]
Fixed already

**Pull request closed without merging**